src/operator/roi_pooling.cc

```diff
@@ -209,13 +207,21 @@ namespace mxnet {
 namespace op {
 
 template<>
-Operator* CreateOp<cpu>(ROIPoolingParam param) {
-  return new ROIPoolingOp<cpu>(param);
+Operator *CreateOp<cpu>(ROIPoolingParam param, int dtype) {
+  Operator* op = NULL;
+  MSHADOW_REAL_TYPE_SWITCH(dtype, DType, {
+    op = new ROIPoolingOp<cpu, DType>(param);
+  });
+  return op;
 }
```

Comment:
It looks good to me. I'll add some testing code for pooling layer later. After that u can follow me to create a testing script to validate ROI pooling result. 
It would be better if u can add `;` after the `}` to make it consistant with other operators.  Many thanks.  ;)